core/eolearn/tests/test_utils/test_common.py

```diff
@@ -50,3 +54,97 @@ def test_is_discrete_type(number_type, is_discrete):
         numpy_dtype = np.dtype(number_type)
 
     assert is_discrete_type(numpy_dtype) is is_discrete
+
+
+@dataclasses.dataclass
+class ApplyToAxesTestCase:
+    function: Callable[[np.ndarray], np.ndarray]
+    data: np.ndarray
+    spatial_axes: Tuple[int, int]
+    expected: Optional[np.ndarray] = None
+
+
+APPLY_TO_TEST_CASES = [
+    ApplyToAxesTestCase(
+        function=partial(np.resize, new_shape=(3, 2)),
+        data=np.ones((2, 3, 4, 1)),
+        spatial_axes=(0, 1),
+        expected=np.ones((3, 2, 4, 1)),
+    ),
+    ApplyToAxesTestCase(
+        function=partial(np.resize, new_shape=(5, 6)),
+        data=np.ones((2, 3, 4, 1)),
+        spatial_axes=(0, 2),
+        expected=np.ones((5, 3, 6, 1)),
+    ),
+    ApplyToAxesTestCase(
+        function=partial(np.resize, new_shape=(2, 4)),
+        data=np.ones((2, 3, 4, 1)),
```

Comment:
just noticed that all data has ndim=4, adjust some test cases

Reply:
add 2, 3 and 5 dim cases